nototools/update_udhr_samples.py

```diff
@@ -76,7 +76,7 @@ def update_udhr(udhr_dir, fetch_dir, in_repo):
 
 def parse_index(src_dir):
   """Parse the index.xml file in src_dir and return a map from bcp to a set of
-  file codes.
+  file codes, and a map from file code to ohchr code.
```

Comment:
what's ohchr ?   I can't find that attribute in xml files. I must be mssing something.

Reply:
It's in the index.xml file.